packages/circuit-ui/components/SelectorGroup/SelectorGroup.tsx

```diff
@@ -122,42 +135,42 @@ export const SelectorGroup = forwardRef(
       multiple,
       size,
       stretch,
+      hideLabel,
       ...rest
     }: SelectorGroupProps,
     ref: SelectorGroupProps['ref'],
   ) => {
     const name = customName || uniqueId('selector-group_');
 
+    if (!options) {
+      return null;
+    }
+
     return (
-      <StyledSelectorGroup
-        role="group"
-        aria-label={label}
-        ref={ref}
-        stretch={stretch}
-        {...rest}
-      >
-        {options &&
-          options.map(({ children, value, ...optionRest }) => (
-            <OptionItem key={value}>
-              <StyledSelector
-                name={name}
-                onChange={onChange}
-                multiple={multiple}
-                value={value}
-                size={size}
-                noMargin
-                checked={
-                  multiple
-                    ? includes(value, activeValue)
-                    : value === activeValue
-                }
-                {...optionRest}
-              >
-                {children}
-              </StyledSelector>
-            </OptionItem>
-          ))}
-      </StyledSelectorGroup>
+      <Fieldset ref={ref} stretch={stretch} {...rest}>
+        {label && <Legend hideLabel={hideLabel}>{label}</Legend>}
```

Comment:
Same

(this is the places where we should throw as discussed, but let's maybe do this in v4 or something)